lib/compat/wordpress-6.7/block-bindings.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Temporary compatibility code for new functionalitites/changes related to block bindings APIs present in Gutenberg.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Adds the block bindings sources registered in the server to the editor settings.
+ *
+ * This allows to bootstrap them in the editor.
+ *
+ * @param array $settings The block editor settings from the `block_editor_settings_all` filter.
+ * @return array The editor settings including the block bindings sources.
+ */
+function gutenberg_add_server_block_bindings_sources_to_editor_settings( $editor_settings ) {
+	// Check if the sources are already exposed in the editor settings.
+	if ( isset( $editor_settings['blockBindings'] ) ) {
```

Comment:
Should the setting be named `blockBindings` or `blockBindingsSources`? Any thoughts?